Tasks/size.js

```diff
@@ -0,0 +1,15 @@
+// Convert number to file size in bytes, kb, mb, and gb
+
+let size=size=>{
+  if (size === 0) return '0';
+  else {
+    const exp = Math.floor(Math.log(size)/Math.log(1000));
+    if (exp==0) return size + ' byte'
+    else if(exp==1) return Math.round(size / 1000) + ' kb'
+    else if(exp==2) return Math.round(size / 1000000) + ' mb'
+    else if(exp==3) return Math.round(size / 1000000000) + ' gb'
+  }
+}
+
+const result = size(123456)
+console.log(result)
```

Comment:
```suggestion
// Convert number to file size in bytes, kb, mb, and gb

const getSize = (size, kb = 1000, mb = 10 ** 6, gb = 10 ** 9) => {
  if (size === 0) return "0";
  else {
    const exp = Math.floor(Math.log(size) / Math.log(1000));
    switch (exp) {
      case 0:
        return size + " byte";
      case 1:
        return Math.round(size / kb) + " kb";
      case 2:
        return Math.round(size / mb) + " mb";
      case 3:
        return Math.round(size / gb) + " gb";
    }
  }
};

const number = 123456;
const result = size(number);
console.log(result);

```
Improve _naming_, _identation_, get rid of _magical numbers_, change elif-s to _switch-case_, add _semicolon_